Update breaking change notes with links to common storage docs

allenporter opened the pull request:
<!--
Before you open the request please review the following guidelines and tips to help it be more easily integrated:

- Describe the scope of your change - i.e. what the change does.
- Describe any known limitations with your change.
- Please run any tests or examples that can exercise your modified code.

Thank you for contributing! We will try to test and integrate the change as soon as we can. There is no need to bump or check in on a pull request (it will clutter the discussion of the request).

Also don't be worried if the request is closed or not integrated sometimes our priorities might not match the priorities of the pull request. Don't fret, the open source community thrives on forks and GitHub makes it easy to keep your changes in a forked repo.
-->

**Description of the change**

Update the changelog to point to the documentation with examples to make it easier to navigate how to resolve breaking changes.

<!-- Describe the scope of your change - i.e. what the change does. -->

**Benefits**

<!-- What benefits will be realized by the code change? -->

It is pretty difficult to navigate breaking changes, especially since the configuration documentation in the readme is hard to map to the old way that volume mounts were handled.


**Possible drawbacks**

<!-- Describe any known limitations with your change -->

Slightly verbose.

**Applicable issues**

<!-- Enter any applicable Issues here (You can reference an issue using #) -->
- fixes #

**Additional information**

<!-- If there's anything else that's important and relevant to your pull request, mention that information here.-->

**Checklist** <!-- [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.] -->
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).

bjw-s commented:
Hi! Thanks for making this PR! I feel that this specific change would be a bit too verbose. 
What I propose instead is that we add a link to the storage docs in the `persistence` sections of the `values.yaml` file. That should make those docs more discoverable.

allenporter commented:
> Hi! Thanks for making this PR! I feel that this specific change would be a bit too verbose.
> What I propose instead is that we add a link to the storage docs in the `persistence` sections of the `values.yaml` file. That should make those docs more discoverable.

I removed the extra docs links in the earlier changelogs and added a [ref] in the docs similar to the other docs that have an external reference.  I left the existing link to docs with a more specific storage link but let me know if that is also too specific. Thanks.

bjw-s commented:
@all-contributors please add @allenporter for documentation
